### DISCUSSION TITLE
Проблемма с GetDocflowEvents

### DISCUSSION THREAD

**Discussion by Mezus47** [diadoc/diadocsdk-1c-docs#946] [General]
Иетод возвращает 405 ошибку , Supported http request methods: POST			

АдресСайта = "diadoc-api.kontur.ru/V3/GetDocflowEvents?boxId="+boxId;
     	
			HTTPЗапрос = Новый HTTPЗапрос;
			HTTPЗапрос.Заголовки.Вставить("POST /V3/GetDocflowEvents");
			HTTPЗапрос.Заголовки.Вставить("Host", "diadoc-api.kontur.ru");
			HTTPЗапрос.Заголовки.Вставить("Accept",		"application/json");  
			HTTPЗапрос.Заголовки.Вставить("Content-Type",	"application/json; charset=utf-8");  
			HTTPЗапрос.Заголовки.Вставить("Authorization", "DiadocAuth ddauth_api_client_id="+Организация.ДиадокAPIКлюч+",ddauth_token="+токен);
			HTTPЗапрос.УстановитьТелоИзСтроки(ЗапросJSON);
			Соединение = Новый HTTPСоединение(АдресСайта,,,, , ,Новый ЗащищенноеСоединениеOpenSSL(),Ложь); 	
			ОтветHTTP = Соединение.Получить(HTTPЗапрос);
			ТекстТЗ =  ОтветHTTP.ПолучитьТелоКакСтроку();

Тело запроса 
{
 "TimeBasedFilter": {
  "FromTimestamp": "2000-01-01T00:00:00",
  "ToTimestamp": "2023-08-28T09:44:33"
 },
 "MessageTypes": "Letter",
 "DocumentDirections": "Inbound",
 "CounteragentBoxId": "// здесь мой BoxId",
 "AfterIndexKey": "null" // и с ним и без него такое, пробывал вообще все убирать из этого тела, результат не меняется
}

**Comment by GilimkhanovDenis** [comment]
Добрый день. вопрос не касается функциональности COM-компоненты

адресуйте запрос в https://github.com/diadoc/diadocapi-docs
